src/components/pager/pager.js

```diff
@@ -22,7 +22,7 @@ class Pager extends React.Component {
     /**
      * Total number of records
      */
-    totalRecords: PropTypes.number.isRequired,
+    totalRecords: PropTypes.string.isRequired,
```

Comment:
Shouldn't we have also defaultProp to totalRecords if we have isRequired in propTypes :)?

Reply:
When a prop isRequired providing a default value breaks the eslint rules we have set up in Carbon `eslint(react/default-props-match-prop-types)`